graphql/endorsement.graphql

```diff
@@ -4,30 +4,29 @@ extend type Query {
     specialEndorsementsForUser(user_id: ID): [SpecialEndorsement]! @field(resolver: "App\\Modules\\Endorsement\\Special\\Mutations\\SpecialEndorsementsForUser")
 }
 
+input SpecialEndorsementInput {
+    name: String! @rules(apply: ["min:3"])
+}
+
+input SpecialEndorsementRequestInput {
+    user_id: ID!
+    endorsement_id: Int! @rules(apply: ["exists:special_endorsements,id"])
+    requested_by: Int!
+}
+
 extend type Mutation {
-    createSpecialEndorsement(
-        name: String! @rules(apply: ["min:3"])
-    ): SpecialEndorsement! @create
+    createSpecialEndorsement(input: SpecialEndorsementInput! @spread): SpecialEndorsement! @create
 
-    updateSpecialEndorsement(
-        id: ID!
-        name: String @rules(apply: ["min:3"])
-    ): SpecialEndorsement @update
+    updateSpecialEndorsement(id: ID!, input: SpecialEndorsementInput! @spread): SpecialEndorsement @update
 
-    deleteSpecialEndorsement(
-        id: ID!
-    ): SpecialEndorsement! @delete
+    deleteSpecialEndorsement(id: ID!): SpecialEndorsement! @delete
 
     grantSpecialEndorsement(
         request_id: ID!
```

Comment:
Use another input type here?

Reply:
There's only one parameter - see the section in the styleguide about this.

Reply:
https://github.com/VATSIM-UK/ukts-core-api/pull/38/files#diff-1e4ce90eff22fd15fd458e414f3d8cccR30-R35

Reply:
Fair enough, just checking for consistency